docs/source/conf.py

```diff
@@ -19,7 +19,7 @@
 #
 import os
 import sys
-import sphinx_rtd_theme
+from metadata import __copyright__, __release__, __version__
```

Comment:
Not sure I follow the logic of this line replacement. Might be easier to see if I knew what issue was resolved.